src/patterns/ec2-app/base.test.ts

```diff
@@ -649,18 +640,14 @@ describe("the GuEC2App pattern", function () {
       accessLogging: { enabled: false },
     });
 
-    const json = SynthUtils.toCloudFormation(stack) as SynthedStack;
-    const lbKey = Object.keys(json.Resources).find(
-      (resource) => json.Resources[resource].Type === "AWS::ElasticLoadBalancingV2::LoadBalancer"
-    );
-    expect(lbKey).toBeTruthy();
-
-    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion -- We assert above that this is truthy
-    const loadBalancer = json.Resources[lbKey!];
-
-    expect(loadBalancer.Properties.LoadBalancerAttributes).toEqual(
-      expect.not.arrayContaining([{ Key: "access_logs.s3.enabled", Value: "true" }])
-    );
+    Template.fromStack(stack).hasResourceProperties("AWS::ElasticLoadBalancingV2::LoadBalancer", {
+      LoadBalancerAttributes: Match.arrayEquals([
```

Comment:
I think this is a simpler, and more complete, test than before. Previously we were checking that `access_logs.s3.enabled` was not present in `LoadBalancerAttributes`; other properties could have been set and changed between changes and we wouldn't know. Now, we're checking the exact contents of `LoadBalancerAttributes`.